lib/access_import/access_import.rb

```diff
@@ -20,8 +20,17 @@ def make_clients
       ).any?
       client_attrs = client_transformer.prepare_attributes(ac_client)
       client = Client.new(client_attrs)
+      client.created_at = ac_client[:d_Rollenbeginn]
+      client.updated_at = ac_client[:d_MutDatum]
       client.user = User.first
-      puts "Importing personen_rolle #{key} to Client.id: #{client.id}" if client.save!
+      next unless client.save!
```

Comment:
Currently the import crashes whenever `save!` is not successful, as it raises an error. The `next unless` doesn't do anything here, as the raised error exits this code.

So either one of the following solutions may apply in this case

1. raise the error and rollback everything when one record fails
1. Check if `client.save` returns `false` and store the failed record in an array and output the failed ones after the whole import process.